src/modules/navigator/mission.cpp

```diff
@@ -629,6 +652,18 @@ Mission::do_need_takeoff()
 bool
 Mission::do_need_move_to_land()
 {
+	if(_mission_item.nav_cmd == NAV_CMD_VTOL_LAND || (_mission_item.nav_cmd == NAV_CMD_LAND && _param_force_vtol.get())){
```

Comment:
This adds a side effect to a check method. I would like to give this it's own work item that would be handled before need to move to land check. So adding it before this line: https://github.com/PX4/Firmware/pull/3720/files#diff-9b5a77642a641b107c8f8a966de34acdR466